opentech/apply/funds/tests/test_views.py

```diff
@@ -205,3 +209,56 @@ def test_get_in_correct_order(self):
             response.context['object_list'],
             [submission.live_revision, revision, revision_older],
         )
+
+
+class TestStaffSealedView(BaseSubmissionViewTestCase):
+    user_factory = StaffFactory
+
+    def test_redirected_to_sealed(self):
+        submission = SealedSubmissionFactory()
+        response = self.get_page(submission)
+        url = self.url_from_pattern('funds:submissions:sealed', kwargs={'pk': submission.id})
+        self.assertRedirects(response, url)
+
+    def test_cant_post_to_sealed(self):
+        submission = SealedSubmissionFactory()
+        response = self.post_page(submission, {'some': 'data'}, 'sealed')
+        # Because of the redirect chain the url returned is not absolute
+        url = self.url_from_pattern('funds:submissions:sealed', kwargs={'pk': submission.id}, absolute=False)
+        self.assertRedirects(response, url)
+
+    def test_non_sealed_unaffected(self):
+        submission = ApplicationSubmissionFactory()
+        response = self.get_page(submission)
+        self.assertEqual(response.status_code, 200)
+
+    def test_non_sealed_redirected_away(self):
+        submission = ApplicationSubmissionFactory()
+        response = self.get_page(submission, 'sealed')
+        url = self.url_from_pattern('funds:submissions:detail', kwargs={'pk': submission.id})
+        self.assertRedirects(response, url)
+
+
+class TestSuperUserSealedView(BaseSubmissionViewTestCase):
+    user_factory = SuperUserFactory
+
+    def test_redirected_to_sealed(self):
+        submission = SealedSubmissionFactory()
+        response = self.get_page(submission)
+        url = self.url_from_pattern('funds:submissions:sealed', kwargs={'pk': submission.id})
+        self.assertRedirects(response, url)
+
+    def test_can_post_to_sealed(self):
+        submission = SealedSubmissionFactory()
+        response = self.post_page(submission, {}, 'sealed')
+        url = self.url_from_pattern('funds:submissions:detail', kwargs={'pk': submission.id})
```

Comment:
can you add a test that the peek was logged?

Reply:
see 2497ab3